src/m365/spo/commands/site/site-apppermission-get.ts

```diff
@@ -0,0 +1,101 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import { SitePermission, SitePermissionIdentitySet } from './SitePermission';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  siteUrl: string;
+  permissionId: string
+}
+
+class SpoSiteApppermissionGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.SITE_APPPERMISSION_GET;
+  }
+
+  public get description(): string {
+    return 'Get a specific application permissions for the site';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.permissionId = typeof args.options.permissionId !== 'undefined';
```

Comment:
Since `permissionId` is required, this will always show as `true` so there is no point in us tracking it.